integration-tests/src/chains/asset_hub_paseo/mod.rs

```diff
@@ -13,27 +15,28 @@ decl_test_parachains! {
 	pub struct AssetHubPaseo {
 		genesis = genesis::genesis(),
 		on_init = {
-			asset_hub_paseo_runtime::AuraExt::on_initialize(1);
+			asset_hub_runtime::AuraExt::on_initialize(1);
 		},
-		runtime = asset_hub_paseo_runtime,
+		// Note: using polkadot as stopgap until paseo updated to polkadot sdk v1.14.0
```

Comment:
`TODO`

Reply:
can actually remove this comment due to the type alias.